app/assets/v2/js/pages/bounty_details.js

```diff
@@ -337,12 +340,30 @@ window.addEventListener('load', function() {
                     actions.push(entry);
                     if(result['status']=='open' && !isBountyOwner(result) ){
                         var entry = {
-                            href: '/funding/claim?source='+result['github_url'],
+                            href: '/_github/auth?redirect_uri=/funding/claim?source=' + result['github_url'],
                             text: 'Claim Issue',
                             parent: 'right_actions',
                             color: 'darkBlue'
                         }
                         actions.push(entry);
+
+                        if (is_interested) {
+                            var entry = {
+                                href: '/uninterested',
+                                text: 'Remove Interest',
+                                parent: 'right_actions',
+                                color: 'darkGrey'
+                            }
+                            actions.push(entry);
+                        } else {
+                            var entry = {
+                                href: '/interested',
+                                text: 'Express Interest',
```

Comment:
seems like if i am not authenticated, this (and remove interest above) should present the user with a dialog asking them to login, no? http://bits.owocki.com/3z0K3v0r3c1c/Screen%20Recording%202017-12-22%20at%2012.26%20PM.gif

Reply:
Whoops!  I'll push a fix for this shortly.